docs/esi-leap-reporting.md

```diff
@@ -5,7 +5,7 @@ of a resource over time.
 
 To generate a csv file with data on all lessees' usage on different nodes:
 ```
-openstack lease contract list --all --long -f csv > report.csv
+openstack lease lease list --all --long -f csv > report.csv
```

Comment:
This particular change results in an awkward command because of the `lease lease` bit. Should we plan on refactoring the CLI somehow?

Reply:
Ah, the actual command would be updated to 'openstack esi lease list'. I'll fix this documentation.